vllm/model_executor/layers/quantization/compressed_tensors/compressed_tensors.py

```diff
@@ -281,15 +286,11 @@ def get_scheme(
         to select the CompressedTensorsScheme used for infernece.
         """
 
-        # Check if the layer is skipped for quantization.
-        # TODO (@robertgshaw2): support module names
-        if should_ignore_layer(layer_name, ignore=self.ignore):
-            return CompressedTensorsUnquantized()
```

Comment:
Is there any reason for UnquantizedLinearMethod vs CompressedTensorsUnquantized?

Reply:
functionally they were doing the same thing. Because the previous lifecycle for compressed-tensors was quant_config --> linear method --> scheme, we were returning a compressed tensors scheme as opposed to a linear method which created weights/applied the forward pass for unquantized layers. Since this PR updates the lifecycle to match the other methods, we no longer need the CompressedTensorsUnquantized and can use the same layer as other quant configs.

Reply:
Thanks for the explanation!